beanstalk_worker/worker_dockerfile_lint.py

```diff
@@ -32,6 +35,37 @@ def write_dockerfile(dockerfile):
         f.write(dockerfile)
 
 
+def export_linter_logs(logs_dir, data):
+    """
+    Export linter logs in given directory
+    """
+    logs_file_path = os.path.join(
+            logs_dir,
+            constants.LINTER_RESULTFILE
+            )
+    logger.log(
+            level=logging.INFO,
+            msg="Linter log file: %s" % logs_file_path
+            )
+    try:
+        fin = open(logs_file_path, "w")
+    except IOError as e:
+        logger.log(
+            level=logging.CRITICAL,
+            msg="Failed to write linter logs on NFS share.")
```

Comment:
I think its the same thing, if the file can't be opened in write mode, we failed to write the logs.